### DISCUSSION TITLE
driver565.57.1  enabled resize-bar，  question：CUDA error at simpleP2P.cu:129 code=205(cudaErrorMapBufferObjectFailed) "cudaDeviceEnablePeerAccess(gpuid[1], 0)"

### DISCUSSION THREAD

**Discussion by liunn5** [NVIDIA/open-gpu-kernel-modules#754] [Q&A]
os：ubuntu 22.04
driver: 565.57.01
card: nvidia 4090D
resize-bar: already  enabled
Problem: SimpleP2p reports the following error:
CUDA error at simpleP2P.cu:129 code=205(cudaErrorMapBufferObjectFailed) "cudaDeviceEnablePeerAccess(gpuid[1], 0)"

**Comment by jason-liu96** [comment]
Hi, I had the same problem, have you solved it yet?
